src/Digdir.Domain.Dialogporten.Domain/Dialogs/Entities/Activities/DialogActivity.cs

```diff
@@ -26,8 +27,10 @@ public class DialogActivity : IImmutableEntity, IAggregateCreatedHandler, IEvent
     [AggregateChild]
     public DialogActivityDescription? Description { get; set; }
 
+    // todo: should PerformedBy still be nullable?
```

Comment:
No, PerformedBy was nullable as a null string indicated that the actor was the service owner.  This is now indicated with an actor instance where the `actorType` is `serviceOwner`.